routes/web.php

```diff
@@ -20,6 +23,11 @@
 
 Route::get('/', static fn () => Inertia::render('Index'));
 
+Route::get('/login', Login::class)->name('login');
+Route::get('/admin/login', static fn () => redirect('/login'));
+Route::get('/logout', Logout::class)->name('logout');
```

Comment:
When I log out from the admin dashboard, I noticed I get redirected back to the UiTID login form. In theory this makes sense because after the logout the user is sent back to the admin page, and a login is required there so they get sent back to UiTID to login. However in reality this will probably be a bit confusing. Is there a way to just send them to a page that confirms that they're logged out and a link to login back again?

And/or, we could create a custom landing page for the admin zone with a login button like the landing page on https://www.uitdatabank.be (when you're not logged in) and show that on `/admin` when not logged in? So then when someone logs out and they see that page again it's probably clear enough that they're logged out. (Optionally we could show a message there confirming that they're logged like in the first suggestion)

Reply:
This required some sorcery 😅

To make this work the default `admin/logout` page needs to be intercepted and execute the `Auth0` logout code.
But this still results in an issue because the default behavior is a redirect to the login page after logout.

The workaround for this issue is to return a response from the logout endpoint, which is a `POST` endpoint. When this response has a `redirect` property the JS code of Laravel Nova will do a redirect to the route specified in this property instead of the login route.

This is the Laravel Nova JS code that handles the logout response
```
    async attempt() {
      if (confirm(this.__('Are you sure you want to log out?'))) {
        this.logout(Nova.config('customLogoutPath'))
          .then(redirect => {
            if (redirect !== null) {
              location.href = redirect
              return
            }

            Nova.redirectToLogin()
          })
          .catch(e => {
            Inertia.reload()
          })
      }
    },
```

Reply:
Looks good!